### PR TITLE
feat(onboarding / privacy): Integrate password validation (zxcvbn lib) in new password screens

### LINKED ISSUE #5096: Integrate password validation (zxcvbn lib) in new password screens
Use new _PrivacyStore_ method `getPasswordStrengthScore` and link it to the new password strength bar value.

### PR DESCRIPTION
**Pull request by noeliaSD**
Closes #5096

### What does the PR do

Used new `PrivacyStore` method `getPasswordStrengthScore` and link it to the new password strength bar value.

Used backend/general to call to `status-go` method and services/general to define the common `GetPasswordStrengthScore` service.

Added onboarding chain to get password strength score information from `OnboardingStore` to `status-go` call.

Fixed `darkmode` issue in onboarding screens.

### Affected areas

Settings Privacy / Change Password
Onboarding / Create New Password

### Screenshot of functionality

- **Settings Privacy / Change Password:**

https://user-images.githubusercontent.com/97019400/159479204-03fe3ce4-3879-4938-9588-9552d8d7e16d.mov

- **Onboarding screens:**

https://user-images.githubusercontent.com/97019400/159679659-10cbaaad-5228-4dd6-a0c7-361f142067f7.mov

### CODE REVIEW

File: ui/app/AppLayouts/Onboarding/stores/OnboardingStore.qml [hunk 1]
```diff
@@ -14,6 +14,11 @@ QtObject {
         onBoardingModul.setSelectedAccountByIndex(selectedAccountIdx)
     }
 
+    function getPasswordStrengthScore(password) {
+        let userName = onBoardingModul.importedAccountAlias
```
Comment (jrainville) [inline ui/app/AppLayouts/Onboarding/stores/OnboardingStore.qml:18]: I know this isn't your fault, but this property name triggers me with its multiple typos.
Maybe we can rename `onBoardingModul` to `onboardingModuleInst` like we do in the other stores?

**Review by jrainville — APPROVED** [review]
Good job. Only aesthetic comments

File: ui/app/AppLayouts/Onboarding/stores/OnboardingStore.qml (hunk 1 above)
Comment (noeliaSD) [reply ui/app/AppLayouts/Onboarding/stores/OnboardingStore.qml:18]: Yeah! I agree! As we are working in the same files with @alexandraB99  regarding the Onboarding flow.. I prefer to keep it here as it is and we can manage it in her's [PR](https://github.com/status-im/status-desktop/pull/4985) for not creating so much conflicts.

**Review by saledjenic — APPROVED** [review]
@noeliaSD it looks good to me, just rebase to the latest master since we removed all interfaces there.

**Review by abhi-status — APPROVED** [review]

**Pull request merged**